test/k6/src/tests/instances.js

```diff
@@ -9,6 +9,8 @@
     -e username=*** `
     -e userpwd=*** `
     -e org=ttd `
+    -e orgNumber=*** `
```

Comment:
let's not split up org, app which are naturally grouped together here. OrgNumber and orgPartyId do not represent the same entity as org. New properties could be placed before org instead?